common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -1338,3 +1342,40 @@ def test_coupon_codes_section_not_visible_for_ecommerce_course(self):
 
         # Log in and visit E-commerce section under Instructor dashboard
         self.assertNotIn(u'Coupon Code List', self.visit_ecommerce_section().get_sections_header_values())
+
+
+class StudentAdminTest(BaseInstructorDashboardTest):
+    SECTION_NAME = 'Test Section 1'
+    SUBSECTION_NAME = 'Test Subsection 1'
+    UNIT_NAME = 'Test Unit 1'
+
+    def setUp(self):
+        super(StudentAdminTest, self).setUp()
+        self.course_fix = CourseFixture(
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run'],
+            self.course_info['display_name']
+        )
+
+        self.vertical = XBlockFixtureDesc('vertical', "Lab Unit")
+
+        self.course_fix.add_children(
+            XBlockFixtureDesc('chapter', self.SECTION_NAME).add_children(
+                XBlockFixtureDesc('sequential', self.SUBSECTION_NAME).add_children(
+                    self.vertical
+                )
+            ),
+        ).install()
+
+        self.username, _ = self.log_in_as_instructor()
+        self.instructor_dashboard_page = self.visit_instructor_dashboard()
+
+    def test_nonrescorable_block(self):
```

Comment:
This test checks the negative case.  Is there an existing test for the positive case?